### PR TITLE
ZEPPELIN-508 Interpreter binding cancel button doesn't work

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Currently, when there is no change compared with before the interpreter binding setting status, the  cancel button in binding page doesn't work. So I fixed this bug.

### What type of PR is it?
Bug Fix

### Todos
* [x] - Fix the cancel button in interpreter binding page.

### Is there a relevant Jira issue?
[https://issues.apache.org/jira/browse/ZEPPELIN-508](https://issues.apache.org/jira/browse/ZEPPELIN-508)

### How should this be tested?
Open the Zeppelin any notebook page -> Click the interpreter binding button 
1. Don't change any binding setting and Just click the **cancel** button. 
2. Or just click twice the interpreter binding button.

### Screenshots (if appropriate)
After this PR applied,
zeppelin508_after 1

### Questions:
* Does the licenses files need update? No.
* Is there breaking changes for older versions? No.
* Does this needs documentation? No.

### CODE REVIEW

File: zeppelin-web/src/app/notebook/notebook.controller.js [hunk 1]
```diff
@@ -509,6 +509,19 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl', function($scope, $ro
...
+      message : 'Nothing is changed.',
+      callback: function(result) {
+        if (result) {
+          $scope.$apply(function () {
+            $scope.showSetting = false;
+            });
+          }
+        }
+      });
+    }
```
Comment (minahlee) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: Just `$scope.showSetting = false;` wouldn't be enough?
Personally I wouldn't expect alert message in this case.
And after clicking cancel, having 'cancel' and 'ok' button choice feels a bit awkward for me.

File: zeppelin-web/src/app/notebook/notebook.controller.js [hunk 2]
```diff
@@ -499,7 +499,7 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl', function($scope, $ro
     if (isSettingDirty()) {
       BootstrapDialog.confirm({
         title: '',
-        message: 'Changes will be discarded',
+        message : 'Changes will be discarded.',
```
Comment (corneadoug) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: There is no space between `message` and `:`

File: zeppelin-web/src/app/notebook/notebook.controller.js (hunk 1 above)
Comment (corneadoug) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: I agree with Mina, no need for Dialog here

**Comment by AhyoungRyu** [comment]
Thanks for the review @minahlee @corneadoug. 
I applied the reviews. : )

**Comment by Leemoonsoo** [comment]
Thanks for the fix.
LGTM. I'm merging it into master.

**Pull request closed without merging**